css/types/counters.json

```diff
@@ -0,0 +1,58 @@
+{
+  "css": {
+    "types": {
+      "attr": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/counters",
+          "description": "<code>counters()</code>",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": true
```

Comment:
Since we no longer allow Firefox CSS values to be true, let's try to track down the exact version!